README.md

```diff
@@ -168,6 +168,7 @@ Command design pattern. You can also use the `FailoverOnMissContainer` decorator
 
 * [Aura.Di Container](https://github.com/auraphp/Aura.Di/blob/develop/src/Aura/Di/ContainerInterface.php)
 * [Guzzle Service Builder](https://github.com/guzzle/service/blob/master/Builder/ServiceBuilderInterface.php)
+* [League Container](https://github.com/thephpleague/container/blob/master/src/ContainerInterface.php)
```

Comment:
@inverse 

In order to keep these sorted alphabetically, I think this line should be moved one down, what do you think?